### DISCUSSION TITLE
Token bridge with quorum based permissioned blockchains

### DISCUSSION THREAD

**Discussion by archm49** [ConsenSys/quorum#1365] [Q&A] [answered]
Are there known issues/limitations with using token bridge in quorum based permissioned blockchains?

**Comment by antonydenyer** [comment, marked as answer]
The main issue is that there's no gas in GoQuorum networks. Consequently if you open your network up to the world you're susceptible to denial of service attacks. But if you're keeping the network permissioned, then there should be no issues.
